client/blocks/product-purchase-features/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -13,15 +13,21 @@ import { localize } from 'i18n-calypso';
 import PurchaseDetail from 'components/purchase-detail';
 import paths from 'lib/paths';
 
-export default localize( ( { selectedSite, translate } ) => {
+export default localize( ( { selectedSite, plan, translate } ) => {
+	const storageLimit =
+		plan === 'business'
```

Comment:
Can we use the `PLAN_BUSINESS` constant instead of `business` string? You can import it with:

```
import {	PLAN_BUSINESS } from 'lib/plans/constants';
```